libs/application/templates/inao/financial-statement-cemetery/src/fields/CemeteryOverview/index.tsx

```diff
@@ -309,7 +326,10 @@ export const CemeteryOverview = ({
       ) : null}
       {fileName ? (
         <>
-          <FileValueLine label={answers.attachments?.file?.[0]?.name} />
+          <FileValueLine
+            label={answers.attachments?.file?.[0]?.name}
+            files={m.files}
```

Comment:
CodeRabbit has some valid concerns here, for example this; shouldn't it be the other way around?
Also other things Coderabbit mentioned